### DISCUSSION TITLE
Feature request for q-tree dense property.

### DISCUSSION THREAD

**Discussion by Claus1** [quasarframework/quasar#10559] [CLI - PWA mode] [answered]
Many of popular elements have dense design option. but q-tree not, that is why the app looks weird if only q-tree non-dense and others are dense. Is it possible to fix it?

**Comment by ontwikkelfabriek** [comment]
@Claus1 

This is the discussions forum section. I would create a Feature Request in the **Issues** section for this feature. 

btw Yes I think it makes a lot of sense for a dense property on the Q-tree.

**Comment by mfoitzik** [comment]
Hi Claus1, I built a Quasar Tree Configurator. You can view it at:
[https://www.simplicityblocks.com/quasartreeconfigurator.html](https://www.simplicityblocks.com/quasartreeconfigurator.html)
I know this is not exactly what you asked for, but I built it for myself when trying to figure out how to create a "denser" tree. It just basically helps you create the override code needed. This was originally done for version 1 and I have not yet tried it with version 2.

**Comment by rstoenescu** [comment, marked as answer]
The reason why QTree does not have a dense mode is that the component is already pretty compact.
But nevertheless, added "dense" prop for an "ultra-compact" mode.

Will be available in Quasar v2.2.4.
PS: this should indeed have been reported as a feature request (so regular ticket instead of using the forum)
